src/views/Send/SendNFT.vue

```diff
@@ -613,7 +615,16 @@ export default {
       this.sendFees = sendFees
     },
     updateSendFees: _.debounce(async function (amount) {
-      await this._updateSendFees(amount)
+      // In NFT screen the balance is equal to the native asset balance
+      console.log(
+        `on updateSendFees => amount: ${amount}, balance: ${this.balance}, asset: ${this.asset}, `
+      )
```

Comment:
@bradleySuira I think we forgot to remove this console log

Reply:
we will keep this for debugging until we test this a lot